test/vue/util.ts

```diff
@@ -33,3 +35,41 @@ export async function ssrRenderOptionsHead(input: any) {
 
   return renderSSRHead(head)
 }
+
+type InstanceType<V> = V extends { new (...arg: any[]): infer X } ? X : never
+type VM<V> = InstanceType<V> & { unmount(): void }
+
+export function mount<V extends Component>(Comp: V) {
+  const el = document.createElement('div')
+  const app = createApp(Comp)
+
+  const unmount = () => app.unmount()
+  const comp = app.mount(el) as any as VM<V>
+  comp.unmount = unmount
+  return comp
+}
```

Comment:
This unit test has to be configured with `environment: 'jsdom'` to work, but this causes problems with some other unit tests, and I need to look into how to turn on the `environment: 'jsdom'` option for this file alone or some other option.